client/css/style.css

```diff
@@ -2286,8 +2286,8 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 		right: 0;
 		background: rgba(0, 0, 0, 0.5);
 		opacity: 0;
-		display: none;
-		transition: opacity 160ms;
+		visibility: hidden;
+		transition: opacity 160ms, visibility 160ms;
```

Comment:
Have to use `visibility` to correctly animate the overlay, as transition doesn't apply on `display` and that was causing a flash.